src/assets/travel/travel_billboard_top.svg

```diff
@@ -0,0 +1,96 @@
+<svg width="968" height="101" viewBox="0 0 968 75" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
This asset is in the `travel` folder, so you could just call it `billboard_top.svg`